src/server/scripts/Kalimdor/zone_orgrimmar.cpp

```diff
@@ -263,8 +264,229 @@ class npc_thrall_warchief : public CreatureScript
 
 };
 
```

Comment:
Oh, right. If you want to comment out part of a script or a whole script to keep it from being part of the Travis compile, then it is correct if you add /* before the script starts and */ only at the end.